diff --git a/dist/aframe-master.js b/dist/aframe-master.js
index bdbc51f4..91a9f7b3 100644
--- a/dist/aframe-master.js
+++ b/dist/aframe-master.js
@@ -79472,7 +79472,7 @@ module.exports.AScene = registerElement('a-scene', {
             }
             navigator.xr.requestSession(useAR ? 'immersive-ar' : 'immersive-vr', {
               requiredFeatures: ['local-floor'],
-              optionalFeatures: ['bounded-floor']
+              optionalFeatures: ['bounded-floor', 'computerVision']
             }).then(function requestSuccess (xrSession) {
               self.xrSession = xrSession;
               vrManager.setSession(xrSession);
@@ -79481,6 +79481,7 @@ module.exports.AScene = registerElement('a-scene', {
                 self.addState('ar-mode');
               }
               enterVRSuccess();
+              window.globals.onEnterXR(useAR ? 'ar' : 'vr');
             });
           } else {
             vrDisplay = utils.device.getVRDisplay();
